src/widgets/modals/VaccinationEvent.js

```diff
@@ -46,18 +51,36 @@ export const NoPCDForm = () => (
 );
 
 export const VaccinationEventComponent = ({
+  editTransaction,
   patient,
-  supplementalDataSchema,
-  vaccinationEventId,
-  vaccinationEventSchema,
   savePCDForm,
   saveSupplementalData,
+  supplementalDataSchema,
   surveySchema,
+  vaccinationEventId,
+  vaccinationEventSchema,
+  vaccines,
 }) => {
   const pcdFormRef = useRef(null);
   const supplementalFormRef = useRef(null);
   const vaccinationFormRef = useRef(null);
 
+  const vaccinationEventNameNote = UIDatabase.get('NameNote', vaccinationEventId);
+  const vaccinationEvent = vaccinationEventNameNote.data;
```

Comment:
Getting this error while testing in emulator.

![Screen Shot 2022-07-01 at 11 59 14 AM](https://user-images.githubusercontent.com/683452/176835258-3018f913-c259-4b29-b357-466e05bd769b.jpg)

Reply:
Some other places this shows up too. Basically we need to validate notes for cases where there is no fields.

Reply:
![Screen Shot 2022-07-01 at 12 27 49 PM](https://user-images.githubusercontent.com/683452/176839042-6c3ec255-95bb-45dc-93b4-afaeb368b775.jpg)